tests/network/conftest.py

```diff
@@ -100,6 +100,11 @@ def istio_system_namespace(admin_client):
     return Namespace(name=ISTIO_SYSTEM_DEFAULT_NS, client=admin_client).exists
 
 
+@pytest.fixture(scope="session")
```

Comment:
https://github.com/RedHatQE/openshift-virtualization-tests/pull/379/files#diff-b1be3011e49a9ad9c396ea72c9895f6fa0e7b23ee1ed84204ac58c4398e95f01R15

Please add this to network_sanity, like virt team did. Network special infra tests should not even begin running, if the cluster does not have everything it needs.

Reply:
I will try to do solve it in a different manner with the help of the test collector and a generic sanity validator.
In the meantime, I suggest to let this in as it does solve the problem of debugging why something failed due to a missing cluster pre-requirement.